### PR TITLE
[RA2 Ch3]: Adding hugepage considerations

### LINKED ISSUE #978: [RA2] Ch3 - add note about KEPs re. NUMA huge pages
From Prague deep dive:
Gerg0 to add a note about the KEP-s under construction for NUMA aware huge pages

### PR DESCRIPTION
**Pull request by CsatariGergely**
Fixes #978 together with #1147

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter03.md [hunk 1]
```diff
@@ -67,10 +67,12 @@ A key thing to note is that the container runtime itself is also a set of proces
 <a name="3.2.1.1"></a>
 #### 3.2.1.1 Memory management
 
-> This chapter should describe considerations about memory management, like huge pages.
+> This chapter should describe considerations about memory management, like hugepages.
 
 > Relate back to features described in the RM [here](../../../ref_model/chapters/chapter05.md#521-virtual-compute). Note that the RM appears to be missing Memory-based HW profile features [here](../../../ref_model/chapters/chapter05.md#54-nfvi-hw-profiles-features-and-requirements).
 
+The Reference Model requires the support of hugepages in `nfvi.com.cfg.004` what is supported by upstream Kubernetes already. In case of some applications the hugepages should be allocated with the considerations of the HW topology. This later feature is missing from Kubernetes, therefore a gap was added to [Chapter 8.2.8](./chapter08.md/#8.2.8)
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter03.md]: nitpick: "in `nfvi.com.cfg.004` what is" > "in `nfvi.com.cfg.004`, which is"

**Comment by CsatariGergely** [comment]
> hugepages

Let's use "Huge Pages" here now. There are some other occurences of "hugepages" and Hugepages elsewhere. I will create tickets and fix those also.

**Comment by CsatariGergely** [comment]
I've opened #1200 to address the hugepage issues.

File: doc/ref_arch/kubernetes/chapters/chapter03.md (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter03.md]: Done

**Pull request merged**